src/main/java/core/basesyntax/randomizer/FigureSupplier.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax.randomizer;
+
+import core.basesyntax.figures.Circle;
+import core.basesyntax.figures.IsoscelesTrapezoid;
+import core.basesyntax.figures.Rectangle;
+import core.basesyntax.figures.RightTriangle;
+import core.basesyntax.figures.Square;
+import core.basesyntax.model.Colors;
+import core.basesyntax.model.Figure;
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int MAX_VALUE = 50;
+
+    public Figure getRandomFigure() {
+        Random random = new Random();
+        ColorSupplier colorSupplier = new ColorSupplier();
```

Comment:
And here I would recommend to make `random` and `colorSupplier` class members. It can be beneficial for several reasons, especially we don not need to create both instances each time method is called.

Reply:
No problem! But when I use "mvn clean package", maven says that I need to do them private. Idk why(

Reply:
@HovorukhaBohdan feel free to make them private! That is even more than best practice, because generally it improves code encapsulation and safety.